backend/src/bundles/users/user.controller.ts

```diff
@@ -10,6 +10,11 @@ import { UsersApiPath } from './enums/enums.js';
 /**
  * @swagger
  * components:
+ *    securitySchemes:
```

Comment:
I am not sure that we have to duplicate it here, this schema was already declared in auth controller

Reply:
You are correct, there was no need for it